crates/lib/src/lib.rs

```diff
@@ -1,8 +1,8 @@
-#![deny(clippy::all)]
-#![deny(clippy::pedantic)]
+//#![deny(clippy::all)]
+//#![deny(clippy::pedantic)]
 #![deny(clippy::cargo)]
 #![allow(clippy::multiple_crate_versions)] // This should be enforced by cargo-deny
-#![deny(missing_docs)]
+//#![deny(missing_docs)]
```

Comment:
Are we able to reintroduce these now?

Reply:
I enabled the pedantic lints too. That was a rabbit hole...